web-frontend/src/pages/TransferCoins/SendCoins/SendConfirmation.tsx

```diff
@@ -31,8 +35,8 @@ interface SendConfirmationProps {
 
 export function SendConfirmation(props: SendConfirmationProps) {
   const { data, handleConfirm, isLoading } = props;
-  const { nickname } = useParams();
-  const { isMainnet } = useFTTransfer(nickname || '');
+
+  const { isMainnet } = useFTTransfer();
```

Comment:
each time you call the useFTTransfer hook it will do http call to station. Be careful about perfomance

Reply:
for example: maybe pass isMainnet in props to reduce the component that call this hook

Reply:
i added the isMainnet variable in the usePrepareScCall hook (used to know which contract to use), i will replace the transferFT hook by this